src/yaml_geometry.jl

```diff
@@ -0,0 +1,301 @@
+# Data structures for YAML wing geometry
+@with_kw struct WingAirfoilInfo
+    csv_file_path::String
+end
+
+@with_kw struct WingSectionData
+    airfoil_id::Int64
+    LE_x::Float64
+    LE_y::Float64
+    LE_z::Float64
+    TE_x::Float64
+    TE_y::Float64
+    TE_z::Float64
+end
+
+@with_kw struct WingAirfoilData
+    airfoil_id::Int64
+    type::String
+    info_dict::WingAirfoilInfo
+end
+
+"""
+    load_polar_data(csv_file_path::String) -> Tuple{Union{Nothing, Tuple{Vector{Float64}, Vector{Float64}, Vector{Float64}, Vector{Float64}}}, Symbol}
```

Comment:
If you return such a complicated structure, isn't it better to define a struct type for it and just document that you return a PolarData struct or how ever you call it?